src/presentation/api/controller/servicesController.go

```diff
@@ -21,7 +35,7 @@ import (
 // @Produce      json
 // @Success      200 {array} dto.ServiceWithMetrics
 // @Router       /services/ [get]
-func GetServicesController(c echo.Context) error {
+func (controller ServicesController) GetServices(c echo.Context) error {
```

Comment:
Faltando o * (em todos) e já que ele pertence à uma struct agora, use apenas Get (eu uso Read agora btw) ao invés de GetServices, o sufixo é inútil (em todos aqui tbm).

Reply:
Ajustado!
Troquei a rapaziada pra Read também, só não mexi nos UC porque fica estranho.